### ISSUE TITLE
TripletInBatchNegSoftmaxContrastLoss的forward函数中的labels有问题

### ISSUE THREAD

**Issue by zhouzj1610** [wangyuxinwhy/uniem#99]
### 🐛 bug 说明

TripletInBatchNegSoftmaxContrastLoss的forward函数中，labels如下：
labels = torch.zeros(sim_matrix.size(0), dtype=torch.long, device=sim_matrix.device)
loss = torch.nn.CrossEntropyLoss()(sim_matrix, labels)

为什么是torch.zeros，是不是应该是：
labels = torch.arange(sim_matrix.size(0), dtype=torch.long, device=sim_matrix.device)

### Python Version

None

**Comment by wangyuxinwhy** [comment]
因为正例都是在 0 索引的位置，所以是 zeros
